kit/dapp/src/app/[locale]/(private)/assets/[assettype]/[address]/regulations/mica/_components/documentation/documents-table-columns.tsx

```
@@ -0,0 +1,436 @@
+      return (
+        <Badge className="bg-success text-success-foreground">
+          <Check className="h-3 w-3 mr-1" /> Approved
+        </Badge>
+      );
+    case "rejected":
+      return (
+        <Badge variant="destructive">
+          <X className="h-3 w-3 mr-1" /> Rejected
+        </Badge>
+      );
+    case "pending":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)